### ISSUE TITLE
Windows service unable to start

### ISSUE THREAD

**Issue by mark-prins** [openmsupply/open-msupply#1636]
## What went wrong? 😲
 
If you screw up the config then the service enters a 'starting' state and cannot be controlled.
You have to kill the process or restart the server to try again.

To kill:

`sc queryex omsupply-server`

make a note of the pid

`taskkill /f /pid [pid]`

## Expected behaviour 🤔
Service should log an event (this happens now)
and enter the 'stopped' state

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Edit the `local.yaml`
2. Do something daft. e.g. put a space in front of `database:`
3. Restart the service

## Your environment 🌱
 - Platform windows server

**Comment by andreievg** [comment]
@mark-prins do you think we can close this one ?

**Comment by mark-prins** [comment]
no, nothing has been done to address it - so (without testing) I expect that it is still an issue.

**Comment by roxy-dao** [comment]
Does anyone know if this is still happening? 👀
